creating/export-blender-to-aframe/index.md

```diff
@@ -18,3 +20,154 @@ Using [A-Frame](https://aframe.io/), we will show you how to export a Blender mo
 ## Finding a model on Sketchfab
 
 ## View the demo
+
+In this guide, we’ll cook up an A-Frame scene containing a model from SketchFab. I chose a model of a Rift CV1 by Ti Kawamoto. It’s a nice model created with Blender, that goes with the theme of WebVR. Plus, the thought of recursive “VR within VR” is amusing..
+For this lesson, I’m using Windows because that is what most VR devs use, but do please reach out to me if you have any questions about doing this in another OS! If you are comfortable setting up a site on a local dev server, feel free to ignore the parts about setting up http-server.
+
+## Instructions for: 
+one simple, locally hosted WebVR site with .dae and .obj versions of a Rift CV1 3D model.
+
+## Ingredients:
+Blender 
+Rift CV1 .blend file from a creator on SketchFab
+9 .jpg image textures that come with this .blend file
+We will create:
+2 model files (.dae and .obj)
+1 texture mapping “material library” file (.mtl to go with the .obj)
+2 directories
+Local web server (we’ll use http-server)
+Index.html file
+
+
+# PART 1: GETTING THE “INGREDIENTS”
+---
+
+## Get Blender
+
+![Image of Blender Home Page](https://github.com/WebVRRocks/webvrrocks/tree/master/creating/export-blender-to-aframe/img/BlenderSite.png)
+
+Go to https://www.blender.org/ and click on the big blue Download Blender button. You’ll come to the downloads page, where you can choose which installer to download, depending on your operating system. Mac, Windows and GNU/Linux are supported. Choose a mirror to download from (probably one closeby). Once you’ve downloaded the installer, open it, and click through the steps of the installation wizard. On Windows, Blender should end up in C:\Program Files\Blender Foundation\Blender. 
```

Comment:
I feel like this would read better as an ordered list:

0. Go to the [Downloads page on Blender.org](https://www.blender.org/download/).
0. Download the version of Blender for your platform (Windows, macOS, Linux).
0. Install the Blender application.
    * **Windows:** Open the `.msi` file and follow the install wizard ([read full instructions](https://www.blender.org/manual/getting_started/installing/windows.html)).
    * **macOS:** Double-click on the `.zip` file to uncompress it, drag the `Blender` and `BlenderPlayer` applications to your `Applications` folder ([read full instructions](https://www.blender.org/manual/getting_started/installing/macos.html)).
    * **Linux:** Uncompress the `.zip` file to your desired location, such as `~/software/` or `/usr/local/` ([read full instructions](https://www.blender.org/manual/getting_started/installing/linux.html)).
0. Launch Blender.
    * **Windows:** `C:\Program Files\Blender Foundation\Blender`
    * **macOS:** `~/Applications/Blender.app`
    * **Linux:** `/usr/local/Blender.app`

<hr>

here's the Markdown source:

```md
0. Go to the [Downloads page on Blender.org](https://www.blender.org/download/).
0. Download the version of Blender for your platform (Windows, macOS, Linux).
0. Install the Blender application.
    * **Windows:** Open the `.msi` file and follow the install wizard ([read full instructions](https://www.blender.org/manual/getting_started/installing/windows.html)).
    * **macOS:** Double-click on the `.zip` file to uncompress it, drag the `Blender` and `BlenderPlayer` applications to your `Applications` folder ([read full instructions](https://www.blender.org/manual/getting_started/installing/macos.html)).
    * **Linux:** Uncompress the `.zip` file to your desired location, such as `~/software/` or `/usr/local/` ([read full instructions](https://www.blender.org/manual/getting_started/installing/linux.html)).
0. Launch Blender.
    * **Windows:** `C:\Program Files\Blender Foundation\Blender`
    * **macOS:** `~/Applications/Blender.app`
    * **Linux:** `/usr/local/Blender.app`
```